bin/alluxio-start.sh

```diff
@@ -50,7 +51,20 @@ check_mount_mode() {
   case "$1" in
     Mount);;
     SudoMount);;
-    NoMount);;
+    NoMount)
+      if [[ "${ALLUXIO_RAM_FOLDER}" =~ ^"/dev/shm"\/{0,1}$ ]]; then
+        if [[ $( uname -a) == Darwin* ]]; then
+          # Assuming Max OS X
+          echo "ERROR: tmpFS is only enabled in Linux."
+          echo -e "${USAGE}"
+          exit 1
+        fi
+        echo "Warning: Alluxio is running on tmpfs that supports swapping."
+        echo "Warning: Check vmstat if Alluxio is slow."
+      else
+        echo "WARNING: using NoMount but ALLUXIO_RAM_FOLDER is not set to /dev/shm."
```

Comment:
I have removed the warning to avoid confusion.

ALLUXIO_RAM_FOLDER=/mnt/ramdisk ./bin/alluxio-start.sh all NoMount is not allowed unless ramFS was mounted manually by the user.